src/Control/Monad/Ghc.hs

```diff
@@ -50,7 +58,11 @@ instance MonadCatch m => MonadThrow (GhcT m) where
     throwM = lift . throwM
 
 instance (MonadIO m, MonadCatch m, MonadMask m) => MonadCatch (GhcT m) where
+#if MIN_VERSION_ghc(9,0,0)
+    m `catch` f = GhcT (unGhcT m `catch` (unGhcT . f))
```

Comment:
this change is pretty surprising to me! ghc has always been very conservative about which modules it depends on, but this change implies that ghc is now depending on http://hackage.haskell.org/package/exceptions?

Reply:
That's correct: https://github.com/ghc/ghc/blame/master/compiler/ghc.cabal.in#L73

Reply:
huh, looks like it does!